src/components/dropdown2.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import { AiFillCaretDown } from 'react-icons/ai';
+
+interface DropdownProps {
+  /**
+   * An object with props of an array of Housing Officers, a selected value to indicate which option is selected and a callback function passed into the Dropdown component
+   */
+  name: HousingOfficers[];
+  selected: string;
+  onSelectedChange(): string
```

Comment:
```suggestion
  onSelectedChange: (housingOfficer: string) => void;
```
This is how you declare a type for a call back function.
The brackets declare the arguments it's expecting (i.e. the housingOfficer argument) and then on the right side of the arrow you declare what the return of the function is going to be (in this case, nothing so we can put `void`)